readme.md

```diff
@@ -39,6 +39,13 @@ chrome_options.add_argument("--proxy-server={0}".format(proxy.proxy))
 browser = webdriver.Chrome(chrome_options = chrome_options)
 ```
 
+Before Running Tests
```

Comment:
Possibly. Are you passing in the full path to the binary?

On Mon, Mar 16, 2015 at 11:24 AM, Chiedo John <notifications@github.com>
wrote:

> In readme.md
> <https://github.com/AutomatedTester/browsermob-proxy-py/pull/31#discussion_r26476368>
> :
>
> > @@ -39,6 +39,13 @@ chrome_options.add_argument("--proxy-server={0}".format(proxy.proxy))
> >  browser = webdriver.Chrome(chrome_options = chrome_options)
> >  ```
> >
> > +Before Running Tests
>
> Hmmm... I'm not sure what was going on there for me then. That was the
> only way I was able to get it to work. Maybe something wrong with my setup?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AutomatedTester/browsermob-proxy-py/pull/31/files#r26476368>
> .
>